runbot_language/view/runbot_language_view.xml

```diff
@@ -0,0 +1,66 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Could you [separate the view files by model name](https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#file-naming).

* `views/runbot_repo.xml`
* `views/runbot_build.xml`